src/layouts/ReviewRequest/components/Comments/SendCommentForm.tsx

```diff
@@ -0,0 +1,67 @@
+import { FormControl, HStack } from "@chakra-ui/react"
+import {
+  Button,
```

Comment:
nit: unused

Reply:
Removed in https://github.com/isomerpages/isomercms-frontend/pull/1102/commits/9379039f25a5b431cb85411063bc5576e2fbbbf6!